tests/js_coverage_tool.py

```diff
@@ -1,228 +0,0 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
-
-###############################################################################
-#  Copyright 2013 Kitware Inc.
-#
-#  Licensed under the Apache License, Version 2.0 ( the "License" );
-#  you may not use this file except in compliance with the License.
-#  You may obtain a copy of the License at
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-#  Unless required by applicable law or agreed to in writing, software
-#  distributed under the License is distributed on an "AS IS" BASIS,
-#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#  See the License for the specific language governing permissions and
-#  limitations under the License.
-###############################################################################
-
-"""
-This module is used for managing javascript coverage data output by the tests
-running in the phantom environment. It is used to reset the coverage, as well
-as to combine and report it once all of the tests have been run.
-"""
-
-import argparse
-import collections
-import glob
-import os
-import six
-import sourcemap
```

Comment:
This was the only place we were using sourcemap, so we should remove it from requirements-dev.txt.